### PR TITLE
Cypress Screenshot

### PR DESCRIPTION
**Pull request by jsalcedo82**

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -2,8 +2,9 @@
   "projectId": "Testing Automation",
   "chromeWebSecurity": false,
   "env": {
-    "username": "admin@test.com",
-    "password": "N6.$!tMr-SBNV4N",
-    "baseUrl": "http://localhost:2368/ghost"
+    "version": "v3.42.5",
```
Comment (jandresboyaca) [inline cypress.json]: Pls leave the default values remember that we are using docker

File: cypress.json [hunk 2]
```diff
@@ -2,8 +2,9 @@
   "projectId": "Testing Automation",
```
Comment (jandresboyaca) [inline cypress.json]: we should need create other one to the new version of ghost and test it

**Review by jandresboyaca — CHANGES_REQUESTED** [review]

**Review by alnodp — APPROVED** [review]

File: cypress.json (hunk 1 above)
Comment (alnodp) [reply cypress.json]: the "version" property is used to name the folder to save screenshots, it doesn't have to do directly with the version used in the execution

File: cypress.json (hunk 2 above)
Comment (alnodp) [reply cypress.json]: I'm not sure if we could use different projectId with the same source code.

**Review by jandresboyaca — APPROVED** [review]

**Pull request merged**